workshop-labs/docs/openshift201/01_git_setup.md

```diff
@@ -26,12 +26,12 @@ git push --set-upstream origin [username]-201
 
 ### Create your branch with the GUI
 
-Navigate to the public repository (<https://github.com/{admin}/devops-platform-workshops-labs)>
+Navigate to the public repository (<https://github.com/[username]/devops-platform-workshops-labs)>
 
-- Click "Branch:" and add your [username]-201 as the branch name.
-- Click "Create branch: [username]-201"
+- Click "Branch:" and add you [username]-201 as the branch name.
+- Click "Create branch [username]-201"
 
-Follow the above instructions to "Create your branch with the CLI" but exclude the `git branch` and `git push` commands to clone a local copy and switch to the already created [username]-201 branch
+Follow the above instructions to "Create your branch with the CLI" but exclude the `git branch` and `git push` commands to clone a local copy and switch to the already create [username]-201 branch
```

Comment:
haha I think `created` is correct (not that im challenging your english :P)

Reply:
you should be.